org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/thingTranslator.js

```diff
@@ -0,0 +1,451 @@
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.battery', [
+            battery
+        ]));
+    }
+
+    return {
+        opent2t: {
+            schema: 'org.opent2t.sample.multisensor.superpopular',
+            translator: 'opent2t-translator-com-wink-sensorpod',
+            controlId: providerSchema['object_id']
+        },
+        pi: providerSchema['uuid'],
+        mnmn: defaultValueIfEmpty(providerSchema['device_manufacturer'], "Wink"),
+        mnmo: defaultValueIfEmpty(providerSchema['manufacturer_device_model'], "Sensor (Generic)"),
+        n: providerSchema['name'],
+        rt: ['org.opent2t.sample.multisensor.superpopular'],
+        entities: entities
+    };
+}
+
+// Each device in the platform has is own unique static identifier
+const deviceIds = {
+    'opent2t.d.sensor.acceleration': 'A547EDCF-DB00-4FE3-81A8-867879718471',
+    'opent2t.d.sensor.airquality': 'CD55D8C0-A485-484D-842C-7F3D692FCAA4',
+    'opent2t.d.sensor.atmosphericpressure': '2A6517F8-261D-4FB1-8B0C-BEC531443175',
+    'opent2t.d.sensor.brightnesschange': 'F18D060B-B672-497C-A66B-44D690EC6825',
+    'opent2t.d.sensor.carbondioxide': 'B533FF0D-0B2D-499D-83E0-57DA256DD720',
+    'opent2t.d.sensor.carbonmonoxide': '153F3969-139C-4D48-AA85-2977DCFF7957',
+    'opent2t.d.sensor.contact': 'A125DBD1-1AA7-4F87-BB5A-7149272F3225',
+    'opent2t.d.sensor.combustiblegas': '3C36C983-3508-4B9A-9C97-4933A8C2D531',
+    'opent2t.d.sensor.glassbreak': '1699836C-EBB2-4F59-BFB0-EA9163B0C2CD',
+    'opent2t.d.sensor.humidity': '61033A42-B81A-4C94-B6F3-23C2B3766B13',
+    'opent2t.d.sensor.illuminance': 'EB3FCB5F-65DC-4228-9125-B97222522E11',
+    'opent2t.d.sensor.locked': '563434A2-801F-4A95-8411-4043054C76ED',
+    'opent2t.d.sensor.loudnesschange': '4389B317-9CF4-4DAD-BC18-C061090B88BA',
+    'opent2t.d.sensor.motion': 'DC0EE2A1-B8B2-45FC-A14E-56F8544DBCE8',
+    'opent2t.d.sensor.presence': 'A9FD1BD1-7CCB-4C8F-9270-4F30AEC33886',
+    'opent2t.d.sensor.temperature': '08A956B2-9ED0-413D-9741-75D71A3F168C',
+    'opent2t.d.sensor.uvradiation': '7CF338CB-A0EF-43C9-A2D7-6DFAF8196069',
+    'opent2t.d.sensor.vibrationchange': 'EFB10535-44D4-44A7-AB54-AE1B4BC02AB0',
+    'opent2t.d.sensor.smoke': 'E978737B-1AFD-4B44-BA6D-193D87C5FD1A',
+    'opent2t.d.sensor.touch': '3A04BBFD-8CDF-48B5-9511-42425BB2DBE1',
+    'opent2t.d.sensor.water': '00FC03C5-81D7-48A1-A2E8-3919A9A3D33F',
+    'opent2t.d.battery': 'CD873FF4-79BB-47BD-865A-2D1357601E6A'
+}
+
+// This translator class implements the 'org.opent2t.sample.multisensor.superpopular' interface.
+class Translator {
+
+    constructor(deviceInfo) {
+        console.log('Wink Sensorpod initializing...');
+
+        validateArgumentType(deviceInfo, "deviceInfo", "object");
+       
+        this.controlId = deviceInfo.deviceInfo.opent2t.controlId;
+        this.winkHub = deviceInfo.hub;
+        this.deviceType = 'sensor_pods';
+
+        console.log('Wink Sensorpod initializing...Done');
```

Comment:
>console [](start = 8, length = 7)

These should be ConsoleLogger.info() hwere ConsoleLogger = new OpenT2TLogger(logLevel="info")  (see other translators for guidance)